xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/signing/Signature.java

```diff
@@ -56,4 +69,27 @@ static ImmutableSignature.Builder builder() {
   default String base16Value() {
     return BaseEncoding.base16().encode(value().toByteArray());
   }
+
+  /**
+   * Custom Jackson serializer for {@link Signature}es.
+   */
+  class SignatureSerializer extends JsonSerializer<Signature> {
+    @Override
+    public void serialize(Signature signature, JsonGenerator gen, SerializerProvider provider) throws IOException {
+      gen.writeString(signature.base16Value());
+    }
+  }
+
+  /**
+   * Deserializes signature string value to an object of type {@link Signature}.
+   */
+  class SignatureDeserializer extends JsonDeserializer<Signature> {
+
+    @Override
+    public Signature deserialize(JsonParser jsonParser, DeserializationContext ctxt)
+      throws IOException {
+      JsonNode node = jsonParser.getCodec().readTree(jsonParser);
+      return Signature.builder().value(UnsignedByteArray.fromHex(node.asText())).build();
+    }
+  }
```

Comment:
Let's put these in `xrpl4j-model` in the `jackson.modules` package

Reply:
`Signature` is  a part of the `xrpl4j-crypto-core` module and might introduce circular dependency if we add crypto-core as a dependency in `xrpl4j-model` .